### ISSUE TITLE
Adding a variable of learning rate for each layer

### ISSUE THREAD

**Issue by xialeiliu** [tensorflow/tensorflow#13803]
I am wondering that is it easy to add a variable in tf.get_variable function, which allows us to initialize the learning rate for each layer easily. Since I found out that it's not easy to do in tensorflow but easy to do in other toolboxes.

**Comment by drpngx** [comment]
I think the easiest way is to use the optimizer API to get the gradients. It will be returned as a nested map where you have the variable names on one side, and the gradients on the other. They come pre-multiplied by the global learning rate, and you can re-multiply.

Did you try on stackoverflow? Maybe there's an even simpler way.

**Comment by xialeiliu** [comment]
Thanks for your comments. I found some solutions on stackoverflow, here is the link:
https://stackoverflow.com/questions/34945554/how-to-set-layer-wise-learning-rate-in-tensorflow
I didn't try, but I guess it works in that way. However it does not work like what I would expect.

**Comment by facaiy** [comment]
Hi, @xialeiliu , so this is a feature request, right?

**Comment by xialeiliu** [comment]
Yes, you are right.

**Comment by drpngx** [comment]
What is the feature request? What is wrong with the second answer on stack overflow?

**Comment by xialeiliu** [comment]
I found second answer is convenient to pass the variables needed to optimizer. Sorry for the late reply.
